### DISCUSSION TITLE
Open coc-reference in refactor window?

### DISCUSSION THREAD

**Discussion by esbenboye** [neoclide/coc.nvim#3350] [Q&A]
This is not a feature request, but a question if a given feature exists... 

Every now and then I do a `<Plug>(coc-references)` and it opens in a list preview window. When I scroll through the list, it updates a window with a code-preview. 

I don't think that overview is very helpfull, since I need to see a larger portion of the code to determine if that is what I'm looking for.

Is it possible to configure CoC to open the results from `<Plug>(coc-references)` in a window similar to the window you get when running a `CocSearch`?

**Comment by chemzqm** [comment]
`:h <plug>(coc-refactor)`

**Reply by esbenboye** [reply]
I see where you are going with this, but unfortunately it doesn't help. 

I'm mostly writing php, but the provided language server does not support renaming, so when I do a `<plug>(coc-refactor)`, I'm getting an error.
